lib/shim/specs/message-subscribe.js

```diff
@@ -27,6 +27,13 @@ class MessageSubscribeSpec extends MessageSpec {
    */
   consumer
 
+  /**
+   * Indicates names of functions to be wrapped for message consumption.
+   * This must be used in tandem with consumer.
+   * @type {Array<string>|null}
+   */
+  functions
```

Comment:
The provided documentation is not helping me understand the field. I think an `@example` would go a long way on this one.

```suggestion
  /**
   * Indicates names of functions to be wrapped for message consumption.
   * This must be used in tandem with consumer.
   *
   * @type {Array<string>|null}
   */
  functions
```